splitio/storage/pluggable.py

```diff
@@ -31,6 +35,10 @@ def get(self, split_name):
             return None
         return splits.from_raw(split)
 
+    def _get_key(self, key):
```

Comment:
Do we need this extra method in `PluggableSplitStorage`, when you can call `self._pluggable_adapter.get(key)` directly in line 119 ?

Reply:
will remove